modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/AndesConfigurationManager.java

```diff
@@ -108,46 +133,33 @@
      * @throws AndesException
      */
     public static void initialize(int portOffset) throws AndesException {
+
+        String brokerConfigFilePath = ROOT_CONFIG_FILE_PATH + ROOT_CONFIG_FILE_NAME;
+        log.info("Main andes configuration located at : " + brokerConfigFilePath);
+
         try {
 
             compositeConfiguration = new CompositeConfiguration();
-
             compositeConfiguration.setDelimiterParsingDisabled(true);
-            
-            log.info("Main andes configuration located at : " + ROOT_CONFIG_FILE_PATH + ROOT_CONFIG_FILE_NAME);
 
-            
             XMLConfiguration rootConfiguration = new XMLConfiguration();
             rootConfiguration.setDelimiterParsingDisabled(true);
-            rootConfiguration.setFileName(ROOT_CONFIG_FILE_PATH + ROOT_CONFIG_FILE_NAME);
+            rootConfiguration.setFileName(brokerConfigFilePath);
             rootConfiguration.setExpressionEngine(new XPathExpressionEngine());
             rootConfiguration.load();
             compositeConfiguration.addConfiguration(rootConfiguration);
-            // Load and combine other configurations linked to broker.xml
-            String[] linkedConfigurations = compositeConfiguration.getStringArray("links/link");
-
-            for (String linkedConfigurationPath : linkedConfigurations) {
```

Comment:
Removed this loop since we don't have any linked configurations other than broker.xml. If we plan to refer other configuration files within broker.xml, we can add this later.